hyperspace/core/src/substrate/mod.rs

```diff
@@ -1,16 +1,18 @@
 pub mod macros;
 
-pub mod composable;
 // pub mod dali;
 pub mod default;
-pub mod picasso_kusama;
-pub mod picasso_rococo;
-
-pub use composable::ComposableConfig;
-// pub use dali::DaliConfig;
-pub use default::DefaultConfig;
-pub use picasso_kusama::PicassoKusamaConfig;
-pub use picasso_rococo::PicassoRococoConfig;
+// pub mod picasso_kusama;
+// pub mod picasso_rococo;
+// pub mod composable;
```

Comment:
Uncomment

Reply:
need to regenerate subxt types once we update polkadot version in composable repo.

Reply:
We'll do this after the upgrade